### ISSUE TITLE
Opening up Radial Menu pressing V causes glitch to hotbar

### ISSUE THREAD

**Issue by FluffiKiki** [CAS-ual-TY/Spells-And-Shields#16]
On the latest 1.19.2 version 2.12.3, whenever I open the radial menu by pressing V, it causes the number keys for the hotbar to not react as quickly. In order to switch on-hand items, I would have to press and hold numbers instead of being able to press and switch instantly.

**Comment by FluffiKiki** [comment]
Closing this, it appears to be a lag issue rather than a mod issue as it still persists. It just happened every time I opened the radial menu. Please let me know if you'd like to look into it still.

**Comment by CAS-ual-TY** [comment]
So your entire game lags when you have the radial menu open?

**Comment by FluffiKiki** [comment]
Yes -- the change is subtle, and I don't have an appropriate screen-recording program, but to recreate the event goes like this:
1. Open radial menu by pressing V
2. Once exited, attempt to press numbers to move along hotbar. 
Trying to do so won't allow them to move, however, you just press and hold it. I have a log, in case you notice if something else is causing it. https://paste.ee/p/4sEtk
Note: Although I still have to press and hold numbers, the reaction time is quicker earlier in the game.
